src/reducers/rootReducer.js

```diff
@@ -1,15 +1,11 @@
 import { combineReducers } from "redux";
 import { reducer as formReducer } from "redux-form";
 import {
-  addMarkerReducer,
-  marksFetchReducer,
-  marksToggleReducer
+  marksReducer
```

Comment:
I'm wondering the reason why one is marksReducer (lowercase) and one is ReviewReducer (uppercase). 
we'd better make the same rule.

Reply:
I will fix that